src/bindings/js/node/include/addon.hpp

```diff
@@ -18,6 +18,7 @@ struct AddonData {
     Napi::FunctionReference partial_shape;
     Napi::FunctionReference ppp;
     Napi::FunctionReference tensor;
+    Napi::FunctionReference save_model;
```

Comment:
Thank you for the explanation. Makes sense now! 

Quick question regarding `init_function`. After removing lines related to `FunctionReference` it will contain only this line 

```cpp
Napi::Function::New(env, func, func_name);
```
, so I propose removing `init_function` as well and replace it with the line above inside `init_module`.

Reply:
I forgot that `exports.Set` is also necessary, so I'll keep `init_function`.